lib/merge.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+function merge (defaultObject, overrideObject) {
+  var newObject = JSON.parse(JSON.stringify(defaultObject))
```

Comment:
On big objects, this is a pretty heavy operation, but as the default object being passed in only ever has three properties, that shouldn't be a problem.